openfl/federated/task/runner_keras.py

```diff
@@ -189,13 +198,10 @@ def train_(self, batch_generator, metrics: list = None, **kwargs):
         #  defined) then the model must be recompiled.
         try:
             results = self.model.get_metrics_result()
+            if len(results) == 0:
+                results = self._initialize_metrics_result(kwargs)
```

Comment:
Looking at the def, `_initialize_metrics_result` only requires `batch_size` from `kwargs`.

```python
batch_size = kwargs.get("batch_size", 1)
results = self._initialize_metrics_result(batch_size)  # Need not pass kwargs
```

Reply:
Fixed in latest commit